### PR TITLE
Feat/dev

### PR DESCRIPTION
**Pull request by rodentskie**
test

### CODE REVIEW

**Review by rodentskiie — COMMENTED** [review]
Just a comment in review and not in code.

File: library/go/types/pr.go [hunk 1]
```diff
@@ -33,7 +34,15 @@ type issue struct {
 }
 
 type comment struct {
-	ID      int32           `json:"id"`
+	ID      int             `json:"id"`
```
Comment (rodentskie) [inline library/go/types/pr.go:37]: Why change from `int32` to `int`?

Comment (rodentskiie) [reply library/go/types/pr.go:37]: coz i like it

File: library/go/types/main.go [hunk 2]
```diff
@@ -2,20 +2,20 @@ package types
...
-	Number      int32                 `json:"number"`
+	Number      int                   `json:"number"`
 	PullRequest pullRequest           `json:"pull_request"`
 	Repository  pullRequestRepository `json:"repository"`
 }
 
 type ReviewRequestPullRequest struct {
 	Action            string               `json:"action"`
-	Number            int32                `json:"number"`
+	Number            int                  `json:"number"`
```
Comment (rodentskiie) [inline library/go/types/main.go:18]: Test

**Pull request merged**